aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -32,20 +35,25 @@ namespace native {
 // Define the per-batch functions to be used in the main implementation of the batched
 // linear algebra operations
 template<class scalar_t>
-void lapackGesv(int n, int nrhs, scalar_t* a, int lda, int* ipiv, scalar_t* b, int ldb, int* info) {
+void lapackGesv(int n, int nrhs, scalar_t *a, int lda, int *ipiv, scalar_t *b, int ldb, int *info) {
   AT_ERROR("gesv only takes float or double Tensors");
 }
 
 template<class scalar_t>
-void lapackGetrf(int m, int n, scalar_t* a, int lda, int *ipiv, int *info) {
+void lapackGetrf(int m, int n, scalar_t* a, int lda, int* ipiv, int* info) {
   AT_ERROR("getrf only takes float or double Tensors");
 }
 
 template<class scalar_t>
-void lapackGetri(int n, scalar_t *a, int lda, int *ipiv, scalar_t *work, int lwork, int *info) {
+void lapackGetri(int n, scalar_t* a, int lda, int* ipiv, scalar_t* work, int lwork, int* info) {
```

Comment:
Oh no, the `*` placement doesn't look consistent (it looks like you've been changing these to occur right before the variable name?)

Reply:
The intention was to make it consistent in the entire file. I will revert this if you prefer that.

Reply:
Making it consistent is good, I am just pointing out that there are some other declarations that are not consistent: 
![image](https://user-images.githubusercontent.com/5652049/48079004-7d774280-e1b8-11e8-95d2-02f6b84a08e5.png)
It would be great if you could make all of them consistent

Reply:
Oh yikes, seems like I missed these. I’ll change them too. Thanks!